app/javascript/dashboard/routes/dashboard/settings/automation/EditAutomationRule.vue

```diff
@@ -507,6 +513,15 @@ export default {
       if (type === null) return false;
       return true;
     },
+    getFileName(id, actionType) {
+      if (!id) return '';
+      if (actionType === 'send_attachment') {
+        const file = this.automation.files.find(item => item.blob_id === id);
+        // replace `blob_id.toString()` with file name once api is fixed.
+        if (file) return file.blob_id.toString();
```

Comment:
@tejaswinichile Once file name is added to the files `array`, can you please replace this `blob_id.toString()` with the file name key. Added this temporarily.

Reply:
<img width="809" alt="multiple file attachment" src="https://user-images.githubusercontent.com/10827585/164444518-5c636f75-196e-4c28-b81a-9fa9417be123.png">

@fayazara, I have resolved the filename issue and multiple file uploads not showing up case.